packages/superset-ui-encodeable/test/index.test.ts

```diff
@@ -0,0 +1,5 @@
+describe('My Test', () => {
```

Comment:
also not sure if you still need this one?

Reply:
removed.